Modules/Invoices/Tests/Feature/InvoiceDeletionValidationFeatureTest.php

```
@@ -0,0 +1,293 @@
+
+        /** Act */
+        $response = $this->post(route('invoices.delete', ['invoiceId' => $invoice->invoice_id]));
+
+        /** Assert */
+        $response->assertRedirect(route('invoices.index'));
+        
+        // Verify invoice was deleted despite being sent
+        $this->assertDatabaseMissing('ip_invoices', [
+            'invoice_id' => $invoice->invoice_id,
+        ]);
+    }
```

PHPMD (2.15.0)

The method it_allows_deletion_when_config_enabled is not named in camelCase. (undefined)

(CamelCaseMethodName)